plugin/teksi_wastewater/interlis/utils/various.py

```diff
@@ -36,16 +37,30 @@ def get_pgconf_as_ili_args() -> List[str]:
     """Returns the pgconf as a list of ili2db arguments"""
     pgconf = DatabaseUtils.get_pgconf()
     args = []
-    if pgconf["host"]:
-        args.extend(["--dbhost", '"' + pgconf["host"] + '"'])
-    if pgconf["port"]:
-        args.extend(["--dbport", '"' + pgconf["port"] + '"'])
-    if pgconf["user"]:
-        args.extend(["--dbusr", '"' + pgconf["user"] + '"'])
-    if pgconf["password"]:
-        args.extend(["--dbpwd", '"' + pgconf["password"] + '"'])
-    if pgconf["dbname"]:
-        args.extend(["--dbdatabase", '"' + pgconf["dbname"] + '"'])
+    dbparams = []
+    for key in pgconf:
+        if key == "host":
+            args.extend(["--dbhost", '"' + pgconf["host"] + '"'])
+        elif key == "port":
+            args.extend(["--dbport", '"' + pgconf["port"] + '"'])
+        elif key == "user":
+            args.extend(["--dbusr", '"' + pgconf["user"] + '"'])
+        elif key == "password":
+            args.extend(["--dbpwd", '"' + pgconf["password"] + '"'])
+        elif key == "dbname":
+            args.extend(["--dbdatabase", '"' + pgconf["dbname"] + '"'])
+        else:
+            dbparams.extend([f"{key}={pgconf[key]}"])
+    if dbparams:
+        # write into tempfile and add path to args
+        dbparams_path = os.path.join(tempfile.gettempdir(), str(uuid.uuid4()))
```

Comment:
I think there is an utility function to get an unique temp folder but I can't remember the name yet.

Reply:
casting a uuid to string should have the same effect I think